src/Raven.Server/Documents/Handlers/Debugging/IdentityDebugHandler.cs

```diff
@@ -14,15 +13,10 @@ public Task GetIdentities()
             using (Database.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             using (context.OpenReadTransaction())
             {
-                var record = Database.ServerStore.Cluster.ReadDatabase(context, Database.Name);
-                var identitiesAsJson = record.Identities.ToJson();
-
-                using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
-                {
-                    context.Write(writer, identitiesAsJson);
-                    writer.Flush();
-                }
+                var json = Database.ServerStore.Cluster.Read(context, Constants.Documents.IdentitiesPrefix + Database.Name.ToLowerInvariant());
```

Comment:
This assume a single object holding all identities per database, right?
That is going to be a problem if we'll have users with thousands of identities. Better to use a value per identity per db.
So the key is going to be `db/db-name/identities/identity-name`.
The idea is that deleting the db will also delete the identities.